ksqldb-rest-app/src/test/resources/ksql-plan-schema/schema.json

```diff
@@ -483,6 +483,12 @@
             "type" : "string"
           }
         },
+        "selectedKeys" : {
+          "type" : "array",
+          "items" : {
+            "type" : "string"
+          }
+        },
```

Comment:
Needed to re-generate the schema because we added the new optional field `selectedKeys` -- should be a backward compatible change so it seem ok.